scripts/generate_crds_manifests.sh

```diff
@@ -89,8 +80,89 @@ spec:
       name: quay.io/operator-framework/configmap-operator-registry:latest
 EOF
 
+cat << EOF > manifests/0000_50_olm-06-psm-operator.deployment.yaml
```

Comment:
is there a way this can go into a real file? I'm missing why cat/heredoc is needed (if this was intentional)

Reply:
Nope, no real reason. I can tackle something nicer in a follow-up PR.